heartbeat/Makefile

```diff
@@ -12,9 +12,9 @@ collect: imports kibana
 
 # Generate imports for all monitors
 .PHONY: imports
-imports: python-env
+imports:
 	@mkdir -p include
-	@${PYTHON_ENV}/bin/python ${ES_BEATS}/script/generate_imports.py --out monitors/defaults/default.go ${BEAT_PATH}
```

Comment:
If we are not using this script in any other place we could remove it :slightly_smiling_face:

Reply:
yeap, on its way, just wanted to make sure the CI went green before pushing that too